packages/babel-parser/test/fixtures/experimental/import-attributes-deprecatedAssertKeyword/invalid-escaped-assert/output.json

```diff
@@ -24,7 +24,7 @@
           },
           "value": "foo"
         },
-        "assertions": []
+        "attributes": []
```

Comment:
This test (and the other with a similar change) were meant to test the importAttributes plugin.